x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/type/EsqlDataTypeRegistry.java

```diff
@@ -46,7 +52,7 @@ public DataType fromJava(Object value) {
 
     @Override
     public boolean isUnsupported(DataType type) {
-        return EsqlDataTypes.isUnsupported(type);
+        return type == DataType.UNSUPPORTED;
```

Comment:
Maybe we should just inline this too, while we're at it?  Or in the next PR...

Reply:
It's used in a while bunch of places. Maybe we should, but I don't want to here.